netbout-client/src/main/java/com/netbout/client/RtAttachment.java

```diff
@@ -119,7 +119,17 @@ public boolean unseen() throws IOException {
 
     @Override
     public Date date() throws IOException {
-        return new Date();
+        return new Date(
```

Comment:
@CarlosGines I suggest adding  a puzzle here to remove the duplication of this section here:
```java
         this.request.fetch()
            .as(RestResponse.class)
            .assertStatus(HttpURLConnection.HTTP_OK)
            .as(XmlResponse.class)
            .xml()
            .xpath(this.xpath("ctype/text()"))
            .get(0);
```

It's ok that this got copy and pasted here, in fact anything but doing so would be wrong according to our principles, but we should ensure that this get's cleaned up down the line imo :)